src/masternode-budget.h

```diff
@@ -365,12 +365,12 @@ class CFinalizedBudget
 
     // Verify and vote on finalized budget
     void CheckAndVote();
-    //total pivx paid out by this budget
+    // Total ZENZO paid out by this budget
```

Comment:
I believe this should be:

Total ZNZ paid out by this budget

Reply:
That works, non-blocking nit though, possibly for another PR.